toolchain/parser/parse_node_kind.h

```diff
@@ -5,58 +5,22 @@
 #ifndef CARBON_TOOLCHAIN_PARSER_PARSE_NODE_KIND_H_
 #define CARBON_TOOLCHAIN_PARSER_PARSE_NODE_KIND_H_
 
-#include <cstdint>
```

Comment:
FWIW, this header is still needed for `uint8_t`. But I'm guessing something else happens to include it transitively.

Not suggesting an edit here because it'll block squashing and merging and it doesn't seem super important.

Reply:
The include is provided in enum_base.h, which had allowed similar includes to be removed from other headers. I'd actually assumed this was deliberate (in my changes, it was -- part of standardizing the approach and hiding implementation details)

Reply:
(either way, something I think we can clean up separately, just want to make sure we're consistent cross-use)